src/app/components/address-displayer/address-displayer.layout.tsx

```diff
@@ -8,9 +8,10 @@ export function AddressDisplayerLayout({ isEven, ...props }: AddressDisplayerLay
   return (
     <styled.span
       color={isEven ? 'accent.text-primary' : 'accent.text-subdued'}
-      fontFamily="Fira Code"
-      mr="tight"
+      fontSize="16px"
```

Comment:
Maybe we can add the `fontSize` +  `lineHeight` into the `typography.ts` file? 

`"mono"` is only used here and in the `AppVersionLabel`

Reply:
Yeah, but in the `AppVersionLabel` the font size is different, so I am thinking that is why it was left out of `mono` originally?

Reply:
Going to add more info to the `mono` token and break up into multiple to use with two different sizes.

Reply:
Great stuff 👍